site/.gitignore

```diff
@@ -8,3 +8,4 @@
 # Generated files
 /.tmp/
 /.next/
+/snapshots/
```

Comment:
I guess we can also remove this from `.prettierignore`

Reply:
It would still make sense to have it in prettierignore file for local dx improvements